### ISSUE TITLE
1.6.5 causes infinite loop in submenu activation javascript. Site doesn't fully load.

### ISSUE THREAD

**Issue by jas8522** [brainstormforce/astra#1005]
Site updated to 1.6.5 reaches submenu javascript code, then fails to load the rest of the site while the javascript appears to be in an infinite loop. 
- Tested with Chrome and Firefox -- both hang while running nav JS
- Firefox reports the option to stop the task
- Upon stopping the task, inspector shows:

```
Error: Script terminated by timeout at:
AstraToggleSetup@https://h[SNIPPED]i.ca/wp-content/themes/astra/assets/js/minified/style.min.js?ver=1.6.5:1:3601
@https://h[SNIPPED]i.ca/wp-content/themes/astra/assets/js/minified/style.min.js?ver=1.6.5:1:5128
```

Specifically, it pinpoints this line: `for (t = 0; t < a.length; t++) a[t].addEventListener('click', AstraToggleSubMeu, !1)`

We had to roll the site back to 1.6.4, at which point the issue disappears.

**Comment by Nikschavan** [comment]
Thank you very much for this report @jas8522!

#1006 should fix this problem, Here is the built version of this PR which can be installed on the website - https://cl.ly/acbe8364a4a3

**Comment by Nikschavan** [comment]
This fix was merged and released as v1.6.6 on WordPress.org.

**Comment by jas8522** [comment]
Thanks for the quick fix! Confirmed working great with 1.6.6
